packages/lab/src/input-next/InputNext.tsx

```diff
@@ -173,6 +178,8 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
           [withBaseName("focused")]: !isDisabled && focused,
           [withBaseName("disabled")]: isDisabled,
           [withBaseName("readOnly")]: isReadOnly,
+          [withBaseName(`textAlign${capitalize(textAlignment)}`)]:
```

Comment:
I think this is better done via a different method.

Instead of relying on classes, you can use the CSS variable in styles e.g. https://github.com/jpmorganchase/salt-ds/blob/bf0a790fb8121fb50ee592af965ac0312e814249/packages/core/src/flex-layout/FlexLayout.tsx#L93-L99